app/Models/ArticleTag.php

```diff
@@ -4,12 +4,16 @@
 
 use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Model;
+use Thiagoprz\CompositeKey\HasCompositeKey;
 
 class ArticleTag extends Model
 {
-    use HasFactory;
+    use HasFactory,HasCompositeKey;
```

Comment:
To follow standard recommendation, when you use a trait make each trait in single line. This is not a big problem put it tells a lot when someone know that you follow standard so instead of this 
```php 
 use HasFactory,HasCompositeKey;
```
do this

```php 
 use HasFactory;
 use HasCompositeKey;
``` 

Search for PSR-12 to learn more